chart/kubeapps/values.yaml

```diff
@@ -646,7 +646,7 @@ authProxy:
   image:
     registry: docker.io
     repository: bitnami/oauth2-proxy
-    tag: 5.1.0-debian-10-r24
+    tag: 5.1.1-debian-10-r58
```

Comment:
BTW, can you give a try to 6.x?

Reply:
6.x requires option changes in our chart. I'll try to do that today separately.

Reply:
See #1904